squad/core/migrations/0071_migrate_old_tokens.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.10 on 2018-02-05 17:25
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db.utils import IntegrityError
+from django.template.defaultfilters import slugify
+
+
+def migrate_tokens(apps, schema_editor):
+    OldToken = apps.get_model('core', 'Token')
+    User = apps.get_model('auth', 'User')
+    Token = apps.get_model('authtoken', 'Token')
+
+    for old in OldToken.objects.all():
+        username = slugify(old.description)
+        user = User(username=username)
+        user_group = None
+        if old.project is None:
+            # global token → "staff" user
+            user.is_staff = True
+        else:
+            # project token → user member of the project group
+            group = old.project.group
+            user_group = group.user_groups.create(name='%s-submitters' % group.slug)
```

Comment:
sth is wrong here (save() missing?). When running the migration all resulting tokens are 'global'. They don't have any project/group restrictions and the resulting users don't seem to be assigned to any group. It might be just the data from production is organized this way and we never noticed.